indigo_social/migrations/0008_badgeaward.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.15 on 2024-08-12 11:56
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('indigo_social', '0007_models_i18n'),
+    ]
+
+    replaces = [
+        ('pinax_badges', '0001_initial'),
+    ]
```

Comment:
do we have this here so that the db entries for the old badges get copied over to the new tables, and then in future we would delete both this and the `options` below?

Reply:
this tells django to ignore this migration if the pinax_badges one is already done, but in new installations to use this migration.

The actual data still stays in the old table (and our model is configured to use that table), I'm not changing that for now.

Reply:
got it, thanks!